content/blog/intro-to-mdox/index.md

```diff
@@ -0,0 +1,197 @@
+
+1. Formatting markdown is nice to have as it allows easier editing for future writers. . [Linters](https://github.com/DavidAnson/markdownlint) for markdown that tell you what is wrong do exist, [but fixing must be done manually which is a brittle and time-consuming process](https://github.com/openshift/enhancements/issues/869) as humans do make mistakes. In the end, you don’t want expensive developer time to be spent on ensuring the correct amount of newlines or characters in a single line, do you?
+2. Checking links is another painful process. Link checkers are available but in most cases, they are slow, abuse other websites or are not smart enough to understand your documentation structure.
+3. Code blocks are easy to introduce into docs, but keeping them up-to-date is another problem entirely! [embedmd](https://github.com/campoy/embedmd) alleviates a lot of this by allowing you to copy over content from a file into a code block. But what about command outputs like `--help` commands or something else that needs to be demonstrated, but isn't saved in a file?
+4. Website generators exist, but there are some obstacles. Frontmatter can be really confusing, it needs to be manually crafted for each markdown file. To change one, the writer has to be familiar with the whole project website setup (imagine contributing to such!). Also, in some cases, blank files with just frontmatter need to be added in order to maintain some condition(e.g website sidebar ordering). All those things do not render well on native platforms like GitHub, making it even more confusing and distracting. Furthermore, there is lot’s of magic with links that has to happen. (Things you don’t want to know: Link localization for multi-version docs, relative links change while shifting markdown content from one dir to another, which is a common use-case when dealing with website generators). Ideally, you want all of this to be fully transparent to the documentation contributor or maintainer!
+
+Finally, how many tools are needed to tackle all these problems? Managing configuration for all of them requires having complex shell scripts/Makefile for some unified experience. Additionally, each tool has its own latency and performance issues, leading to huge “bake” times for documentation automation.
+
+## mdox: All in one solution and more!
+
+Fortunately, there is now a way to tackle all these problems (and more) automatically using `mdox`! Great tooling should get out of your way and let you do exactly what you need to without causing pain, which is what we want to achieve with `mdox`.
+
+[`mdox`](https://github.com/bwplotka/mdox) is a simple Go CLI tool, you can install using `go get` or pin using [bingo](https://github.com/bwplotka/bingo).
```

Comment:
worth to add full `go get path`?

Reply:
Yes. Let's add since currently, it is unreleased. Or, we could release it tomorrow?